installers/dhall/check.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -e
+
+CLUSTERS="$(cat $(dirname $0)/../../installer-clusters.cfg | xargs echo -n)"
+OSES="linux macos64 win64"
+
+cd $(dirname $0)
+for os in ${OSES}
+do for cluster in ${CLUSTERS}
+   do echo "checking ${cluster}-${os}"
```

Comment:
I still think this is a bad idea - you have now at two places specified how dhall is composed and you'll have to maintain both of them. Developer will change haskell file and dhall files, CI will fail and they'll have to update this file as well. So what's the benefit?

Reply:
@domenkozar, What would you propose as a better way for a developer to type-check all possible combinations of Dhall files?

Reply:
Initially, the thing was intended for local development, but then someone (maybe you, even) suggested it should be run as part of CI.

Maybe we could backtrack on the CI run, and keep if for local development (where the person doing the development knows what they do..)

Reply:
I fail to see how it's useful for local development, now the whole logic lies in haskell and has nothing to do with shell anymore?

Reply:
@domenkozar, no-no, I mean testing the changes to the Dhall files themselves.

Reply:
I suggest not increasing the scope of this PR, but you can do that if you export haskell functions that do dhall part and them use them through ghci and module reloading.

Reply:
Right, that's a bit heavy-weight. The current shell approach provides a valuable balance, I believe.

Reply:
I would prefer to keep the code DRY if possible, for exactly the reason Domen raised in the initial comment.

Short of that, we should exclude this from CI and add a bold comment near the top of this file to say:

1. it's for illustration purposes only
2. the actual logic for composing the Dhall files is in Haskell

Reply:
This is still not addressed.

Reply:
Ok, I guess I'm confused -- Domen, this was never a part of CI -- now I'm not sure where the idea originated..

Jake, the documentation was added: https://github.com/input-output-hk/daedalus/blob/devops-690-dhall/installers/README.md#generation-of-runtime-configuration-files

Reply:
Following wasn't addressed:

> Short of that, we should exclude this from CI and add a bold comment near the top of this file to say:

> it's for illustration purposes only
> the actual logic for composing the Dhall files is in Haskell

Rodney pointed out that developers now don't have a simple way to generate .yaml files - if we created a separate executable calling `generateConfig` as proposed by Jake to keep things DRY. 

I don't understand why prefer bash script with 14 lines that duplicates behavior is better than 10 lines of haskell just exposing your existing code as something one can run.

Reply:
Ok, will do.